### DISCUSSION TITLE
SSL error with v2.2.5 `(MySQLdb.OperationalError) (2026, 'TLS/SSL error: SSL is required, but the server does not support it')`

### DISCUSSION THREAD

**Discussion by ghilesmeddour** [PyMySQL/mysqlclient#737] [Issue report]
### Describe the bug

Hi,

After bumping `mysqlclient`, I am getting in all my programs following error `(MySQLdb.OperationalError) (2026, 'TLS/SSL error: SSL is required, but the server does not support it')`. Server side, SSL is disabled.
  - Updating mysqlclient (2.2.4 -> 2.2.5)

When downgrading to v2.2.4 everything works fine.

### Environment

- MariaDB 10.3.38
- Windows
- Python 3.12.7

### What version of mysqlclient do you use?

2.2.5

### Minimum but complete code to reproduce

```python
import pandas as pd
import sqlalchemy

db_url = ""
engine = sqlalchemy.create_engine(db_url)
df_tmp = pd.read_sql("SELECT COUNT(*) FROM t", engine)
```

**Comment by methane** [comment]
duplicate of #729.
Try wheels in the thread.
